<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockListDialog</class>
 <widget class="QDialog" name="StockListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Stock</string>
  </property>
  <layout class="QVBoxLayout" name="dialogVerticalLayout">
   <item>
    <widget class="QTableWidget" name="stockTableWidget">
     <property name="font">
      <font>
       <family>Poppins</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>SKU</string>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ISBN</string>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonBarFrame">
     <property name="font">
      <font>
       <family>Poppins</family>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addStockRowButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  border-radius: 12px; background: #3B82F6; color: #FFF; padding: 6px;
}
QPushButton:hover   { background: rgba(59,130,246,200); }
QPushButton:pressed { background: rgba(59,130,246,150); }</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editStockRowButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  border-radius: 12px; background: #FFF; padding: 6px;
}
QPushButton:hover   { background: rgba(255,255,255,200); }
QPushButton:pressed { background: rgba(255,255,255,150); }</string>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteStockRowButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  border-radius: 12px; background: #FFF; padding: 6px;
}
QPushButton:hover   { background: rgba(255,255,255,200); }
QPushButton:pressed { background: rgba(255,255,255,150); }</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpacer">
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelStockButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  border-radius: 12px; background: #FFF; padding: 6px;
}
QPushButton:hover   { background: rgba(255,255,255,200); }
QPushButton:pressed { background: rgba(255,255,255,150); }</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveStockButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  border-radius: 12px; background: #3B82F6; color: #FFF; padding: 6px;
}
QPushButton:hover   { background: rgba(59,130,246,200); }
QPushButton:pressed { background: rgba(59,130,246,150); }</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
